client/blocks/app-promo/style.scss

```diff
@@ -1,46 +1,35 @@
 .app-promo {
-	position: relative;
+	display: flex;
 	max-width: 300px;
-	margin: 0 auto 5px;
-	padding: 5px 0;
+	margin: 10px;
+	padding: 10px;
 	overflow: hidden;
+	background-color: var(--color-surface);
+	box-shadow: 0 0 0 1px rgba(var(--color-neutral-10-rgb), 0.5), 0 1px 2px var(--color-neutral-0);
+	border-radius: 2px;
 
 	@include breakpoint-deprecated( "<660px" ) {
 		display: none;
 	}
 }
 
-.app-promo .app-promo__icon {
-	position: absolute;
-	top: 15px;
-	left: 20px;
-}
-
 .app-promo .app-promo__dismiss {
-	position: absolute;
-	right: 17px;
-	top: 6px;
 	cursor: pointer;
-	padding: 5px 0;
+	padding: 4px;
+	align-self: center;
 
 	svg {
-		fill: var(--color-neutral-10);
+		fill: #616161;
```

Comment:
Any chance to use a color variable here?

Reply:
`#616161` and `#1e1e1e` don't have existing variables associated with them in the Calypso codebase and aren't part of [the Color Studio](https://color-studio.blog/) listing. Would you perhaps suggest still creating variables at the top of this individual stylesheet, or do you think this is fine to overlook in this instance?

Reply:
We can always hardcode a color, but that usually creates problems when the user (or the app) uses a different color scheme that overrides the default colors. Then the hardcoded color might not quite fit into the color scheme and might look odd to the user 😉 Is there a way we can instead pick a variable that is close enough that will do the job?

Reply:
Other buttons with a "cross" icon use `--color-text-subtle`, inherited from `.button.is-borderless`. If there is no strong reason for the App Promo UI to have a truly unique look, it should reuse colors from the surrounding Calypso and the current color theme.

Reply:
Thanks for elaborating further, both. I've gone ahead to replace `#616161` with `--color-text-subtle` (`#2c3338` by default).

Reply:
For future reference, if there are semantic (and similar) colors that exist currently, we should likely always use those colors — often the design doesn't take those into consideration or mistakenly uses hardcoded values.